docs/schema.md

```diff
@@ -356,6 +364,12 @@ The `purl` field is a string following the
  [Package URL specification](https://github.com/package-url/purl-spec) that
 identifies the package. This field is optional but recommended.
 
+The `severity` field is an optional element [defined here](#severity-field). 
+Per package severity override is optional for different packages. It is only 
```

Comment:
Thinking more I'd suggest enforcing that either the top level severity or the per-package severites are set, but not both. 

It doesn't make much sense to me, and it enforces a bit more consistency. We should also validate this with our JSON schema, but that can come later. 

Maybe we can replace everything from the second paragraph onwards with:

```
This `severity` field applies to a specific package, in cases where affected packages have
differing severities for the same vulnerability. If any package level `severity` fields are set,
the top level `severity` (add hyperlink) must not be set. 
```

Reply:
Done! Ready for another review